File: Talleres RESFTUL JAVA DOTNET/API_GOOGLE_MAPS_REST_DOTNET_GR-03/04.SERVIDOR/API-DOTNET-BANK/Application/DTOs/Branch/BranchDto.cs
namespace API_DOTNET_BANK.Application.DTOs.Branch
{
    public class BranchDto
    {
        public int Id { get; set; }
        public string Name { get; set; } = default!;
        public double Latitude { get; set; }
        public double Longitude { get; set; }
        public string PhoneNumber { get; set; } = default!;
        public string Email { get; set; } = default!;
        public bool IsActive { get; set; }
        public DateTime CreatedAt { get; set; }
        public DateTime? UpdatedAt { get; set; }
    }

    public class CreateBranchDto
    {
        public string Name { get; set; } = default!;
        public double Latitude { get; set; }
        public double Longitude { get; set; }
        public string PhoneNumber { get; set; } = default!;
        public string Email { get; set; } = default!;
    }

    public class UpdateBranchDto
    {
        public string Name { get; set; } = default!;
        public double Latitude { get; set; }
        public double Longitude { get; set; }
        public string PhoneNumber { get; set; } = default!;
        public string Email { get; set; } = default!;
        public bool IsActive { get; set; }
    }
}

File: Talleres RESFTUL JAVA DOTNET/API_GOOGLE_MAPS_REST_JAVA_GR-03/04. SERVIDOR/api-java-bank/src/main/java/com/allpasoft/apijavabank/application/dto/BranchDto.java
package com.allpasoft.apijavabank.application.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDateTime;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class BranchDto {
    private Long id;
    private String name;
    private Double latitude;
    private Double longitude;
    private String phoneNumber;
    private String email;
    private Boolean isActive;
    private LocalDateTime createdAt;
    private LocalDateTime updatedAt;
}
